xla/service/gpu/transforms/windowed_einsum_handler_test.cc

```diff
@@ -110,15 +110,15 @@ ENTRY test_main {
   HloInstruction* ag_loop =
       module->entry_computation()->root_instruction()->mutable_operand(0);
   HloComputation* ag_loop_body = ag_loop->while_body();
-  HloInstruction* inst = FindInstructionByName(ag_loop_body, "dot.2");
-  EXPECT_GT(inst->backend_config<GpuBackendConfig>()->operation_queue_id(), 0);
-  EXPECT_TRUE(
-      inst->backend_config<GpuBackendConfig>()->force_earliest_schedule());
-
-  HloInstruction* cp1 =
-      FindInstructionByName(ag_loop_body, "collective-permute");
-  EXPECT_TRUE(
-      cp1->backend_config<GpuBackendConfig>()->force_earliest_schedule());
+  int64_t dot_count = 0;
+  for (HloInstruction* inst : ag_loop_body->MakeInstructionPostOrder()) {
+    if (inst->opcode() == HloOpcode::kDot) {
+      dot_count++;
+      EXPECT_GT(inst->backend_config<GpuBackendConfig>()->operation_queue_id(),
+                0);
+    }
+  }
+  EXPECT_EQ(dot_count, 4);
```

Comment:
You no longer check the `force_earliest_schedule` field on the dots and collective-permute. SHould you check this?

Reply:
good catch, accidentally removed it.